### PR TITLE
add build-tools.sh ALL to github actions, lost with Travis

### PR DESCRIPTION
**Pull request by marc-hb**
2 commits related to recent #4141

### CODE REVIEW

File: .github/workflows/tools.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: User space tools/
```
Comment (cujomalainey) [inline .github/workflows/tools.yml:2]: trailing slash unneeded

File: .github/workflows/tools.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+# 'workflow_dispatch' allows running this workflow manually from the
+# 'Actions' tab
+
+# yamllint disable-line rule:truthy
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  ALL:
```
Comment (cujomalainey) [inline .github/workflows/tools.yml:11]: why caps? why not something more informative such as  "build check"

**Review by cujomalainey — CHANGES_REQUESTED** [review]

File: .github/workflows/tools.yml (hunk 1 above)
Comment (marc-hb) [reply .github/workflows/tools.yml:2]: (too) terse allusion to the `tools/` subdirectory. How about: `User space in tools/`?

File: .github/workflows/tools.yml (hunk 2 above)
Comment (marc-hb) [reply .github/workflows/tools.yml:11]: A bit cryptic allusion to the default CMake target and its reserved name `ALL`, see #4141 for more background.

How about `Build ALL`?

**Comment by lgirdwood** [comment]
Fwiw, looks like an unrelated timeout on internal codec checks (but all green).

**Comment by marc-hb** [comment]
https://sof-ci.01.org/sofpr/PR4210/build9084/devicetest/?model=CML_HEL_RT5682&testcase=verify-pcm-list had the topology panic below that has also been seen elsewhere.

Can't possibly related to building user space tools. I think a recent kernel revert fixed it.

Everything else was green for commit 7f8ebb9b507, especially https://sof-ci.01.org/sofpr/PR4210/build9084/build/tools.txt and https://github.com/thesofproject/sof/pull/4210/checks?check_run_id=2615211180

https://github.com/thesofproject/sof/pull/4210/checks?check_run_id=2643711883 is green again and 
Jenkins CI is down this WE.

```
kernel: [    3.410957] sof-audio-pci-intel-cnl 0000:00:1f.3: ipc tx: 0x30010000: GLB_TPLG_MSG: COMP_NEW
kernel: [    3.411115] sof-audio-pci-intel-cnl 0000:00:1f.3: ipc tx succeeded: 0x30010000: GLB_TPLG_MSG: COMP_NEW
kernel: [    3.411118] sof-audio-pci-intel-cnl 0000:00:1f.3: widget PCM8C setup complete
kernel: [    3.411123] sof-audio-pci-intel-cnl 0000:00:1f.3: ipc tx: 0x30010000: GLB_TPLG_MSG: COMP_NEW
kernel: [    3.411235] sof-audio-pci-intel-cnl 0000:00:1f.3: error : DSP panic!
kernel: [    3.411239] sof-audio-pci-intel-cnl 0000:00:1f.3: panic: dsp_oops_offset 788480 offset 788480
kernel: [    3.411243] sof-audio-pci-intel-cnl 0000:00:1f.3: status: fw entered - code 00000005
kernel: [    3.411402] sof-audio-pci-intel-cnl 0000:00:1f.3: error: assertion failed
kernel: [    3.411405] sof-audio-pci-intel-cnl 0000:00:1f.3: error: trace point 00004000
kernel: [    3.411408] sof-audio-pci-intel-cnl 0000:00:1f.3: error: panic at src/audio/kpb.c:154
kernel: [    3.411411] sof-audio-pci-intel-cnl 0000:00:1f.3: error: DSP Firmware Oops
```

**Review by lgirdwood — APPROVED** [review]
@cujomalainey please merge if you are aligned with the review changes.

File: tools/fuzzer/README.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+This project has been superseded by sof/oss-fuzz/. See also
```
Comment (cujomalainey) [inline tools/fuzzer/README.md:1]: `tools/oss-fuzz`

**Comment by cujomalainey** [comment]
one comment issue otherwise all good

**Comment by marc-hb** [comment]
A couple hardware issues that cannot be related to this.
https://sof-ci.01.org/sofpr/PR4210/build9147/devicetest/?model=BYT_MB_NOCODEC&testcase=check-alsabat-headset-playback has the following alsabat error
```
 FAIL: Peak freq too low 988.77 Hz
 FAIL: Peak freq too high 1004.88 Hz
```
In the same run APL_UP2_HDA was not available for testing or its kernel failed to boot, not sure which one.

Everything else is green including  https://sof-ci.01.org/sof-pr-viewer/#/build/PR4210/build6485167

**Comment by cujomalainey** [comment]
Agreed, unrelated

**Pull request merged**